README.md

```diff
@@ -2,3 +2,4 @@
 for testing out git
 
 [Sohan Rudra](https://github.com/rudrasohan)
+[Vibhakar Mohta](https:github.com/vib2810)
```

Comment:
@vib2810 please check my remarks for #1 and make changes to your file accordingly.